.yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable */
+//prettier-ignore
+module.exports = {
```

Comment:
Just curious about what is this plugin for 🤔

Reply:
To be able to use yarn commands such as `yarn workspaces foreach`.

https://github.com/yarnpkg/berry/blob/HEAD/packages/plugin-workspace-tools/README.md

I had a command using that in this PR but then changed it back. The plugin is anyway useful and we also use it in other repos.